website/content/docs/concepts/architecture.mdx

```diff
@@ -101,6 +101,20 @@ servers they are joined with, but do need to be in the same
 region. Datacenters provide a way to express fault tolerance among jobs as
 well as isolation of infrastructure.
 
+#### Node
+
+A more generic term used to refer to machines running Nomad agents in client
+mode. Unless noted otherwise, it may be used interchangeably with
+[client](#client).
```

Comment:
I kind of disagree with noting that it may be used interchangeably (even though it often is). Node is intended to refer to the machine (and operating system, etc.) while Client refers to the Nomad agent running in client mode. We should avoid conflating the terms in our own docs.

Reply:
> when can it be different?

"Node" is a very generic term that different people use it to mean different things 😅 

For example, a node in a network context can be anything sending or receiving traffic, like a computer, a router, a switch, or maybe even an allocation, like a sidecar proxy. So if you're talking about Nomad's network topology, "node" may not necessarily mean this specific definition.

But to @shoenig's point, I think I didn't express myself well here. I meant more like "you may see or hear people using them interchangeably", which I have in informal conversations, issues, community blog posts etc.

Maybe this sentence would be better like this:

```
Despite being different concepts, you may find the term "node" being used interchangeably with "client" in some materials.
```

> Node is intended to refer to the machine (and operating system, etc.)

To the machine running a Nomad client right? I don't recall us using "node" to refer to machines running servers. For example, the `nomad node` commands and `/v1/node/` endpoints are only about clients.

Reply:
I ping-ponged a bit on how to phrase this and ended up here:
https://github.com/hashicorp/nomad/blob/efc2e3746019c7c62930dab14a02c0377475a79c/website/content/docs/concepts/architecture.mdx#L116-L120

Is this a better description? Or maybe I should just leave this part out?